go-server-server/go/default.go

```diff
@@ -1229,7 +1230,29 @@ func ConfigVrouterVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
             failed = append(failed, r)
             continue
         }
-
+        kv, err := GetKVs(confdb.db_num, generateDBTableKey(confdb.separator, VNET_TB, vnet_id_str))
```

Comment:
Can we have a logic to cache this after getting it first time? We can have significantly high number of routes and for each time, checking the DB can be expensive. So suggest a simple logic to set advertisement field for a vnet.